mbuild/compound.py

```diff
@@ -2353,7 +2353,8 @@ def to_networkx(self, names_only=False):
         Parameters
         ----------
         names_only : bool, optional, default=False Store only the names of the
```

Comment:
```suggestion
        names_only : bool, optional, default=False
        Store only the names of the
```

Small change, but can you put the beginning of this sentence on a new line, rather than directly after the `default=False` section.

Reply:
After this edit, I think this is ready to merge @umesh-timalsina